vagrant/Vagrantfile

```diff
@@ -0,0 +1,29 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box_check_update = false
+
+  config.vm.define "ubuntu" do |instance|
+    instance.vm.box = "ubuntu/xenial64"
+  end
+
+  config.vm.define "centos" do |instance|
+    instance.vm.box = "centos7"
```

Comment:
```suggestion
    instance.vm.box = "centos/7"
```

Reply:
The `centos/7` doesn't have the virtual box extras that we need for "syncing" the folder from host to guest. So thats why I choose the `centos7` box.